--- src/org/apache/tomcat/util/net/jss/JSSSupport.java	2015-04-20 12:34:46.000000000 -0600
+++ src/org/apache/tomcat/util/net/jss/JSSSupport.java	2015-08-05 15:10:53.000000000 -0600
@@ -97,6 +97,10 @@ class JSSSupport implements SSLSupport {
         return null;
     }
 
+    public String getProtocol() throws IOException {
+        return null;
+    }
+
     public String getSessionId() throws IOException {
         return null;
     }
